spec.html

```diff
@@ -12770,33 +12770,33 @@
         </ul>
       </emu-clause>
       <emu-clause id="sec-applying-the-exp-operator">
-        <h1>Applying the `**` operator</h1>
+        <h1>Applying the `**` Operator</h1>
```

Comment:
Actually, it's pretty consistent on using title-case. Your example of sentence-case was changed to title-case 4 days ago in 5ee6f14c. (I guess the rendered version hasn't caught up yet.)

(Looking at some example specs, it seems like ECMA prefers title-case and ISO prefers sentence-case.)